manifests/params.pp

```diff
@@ -33,12 +32,12 @@
 
   case $facts['kernel'] {
     'OpenBSD': { $has_wordexp = false }
-    default:   { $has_wordexp = true }
+    default: { $has_wordexp = true }
```

Comment:
It made sense to align the cases before though?

Reply:
Agreed